src/chapters/aspdotnet-controllers-and-routes/controllers-forms.rst

```diff
@@ -0,0 +1,131 @@
+Controllers with Forms
+======================
+
+Sending Form Data - Video
+-------------------------
+
+.. TODO: Add "Hello ASP.NET Part 4" vid
+
+.. admonition:: Note
+
+   In your forked version of LaunchCode's `HelloASPDotNET <https://github.com/LaunchCodeEducation/HelloASPDotNET>`_, the starter code for this video is on the ``query-parameters`` branch.
+   The solution code for this video is on the ``forms`` branch.
+
+Sending Form Data - Text
+-------------------------
+
+What if we want to send over some form data?
+To send data via a simple form in ASP.NET, we'll set things up like this:
+
+We have an action method that generates a form at index and responds to a ``GET`` request. 
+
+.. sourcecode:: csharp
+   :linenos:
+
+   public IActionResult Index()
+   {
+      string html = "<form method='post' action='Hello/Display'>" +
```

Comment:
one more thing on the forms - we can post to the same address as where the form is displayed. I think it can be a good thing to show them this in addition to showing that the form can be posted to a specific address. this can help distinguish between the getting action (displaying the initial form) and the posting action (displaying the results using the form data)

Reply:
Added a note at the bottom of the page so that students can try this out!